pkg/reconciler/testing/v1/subscription.go

```diff
@@ -138,6 +138,19 @@ func WithSubscriptionSubscriberRef(gvk metav1.GroupVersionKind, name, namespace
 	}
 }
 
+func WithSubscriptionSubscriberRefUsingGroup(gvk metav1.GroupVersionKind, name, namespace string) SubscriptionOption {
```

Comment:
Please add a test that sets both Group and APIVersion. It would be interesting to know what to expect from the reconcilers if I, say, deleted my webhook and set those fields. Am I broken?

Reply:
TBH I don't get what you mean. In any case, is this comment applicable after the change I made here? https://github.com/knative/pkg/pull/2127/commits/ee55b44c12d5256ca312d570df6da5169752ae53#diff-ee7cca2a46e1af5b2aa2a11e1ae0e064e24bf589126d86dc0eb584e0df724966R68